### PR TITLE
feat: command create domains

### PR DESCRIPTION
**Pull request by maxwelbm**
-  command create domains

### CODE REVIEW

File: messages/create/domains/messages.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package domains
+
+var (
+	Usage                    = "domains [flags]"
+	ShortDescription         = "Creates a new domain"
+	LongDescription          = "Creates a Domain based on given attributes to be used in Edge Applications"
+	FlagName                 = "The Domain's name"
+	FlagCnames               = "List of domains' names"
```
Comment (helio-neto) [inline messages/create/domains/messages.go:8]:
```suggestion
	FlagCnames               = "List of domain's names"
```

**Review by helio-neto — APPROVED** [review]

File: messages/create/domains/messages.go (hunk 1 above)
Comment (PatrickMenoti) [reply messages/create/domains/messages.go]: domains' is correct @helio-neto

Comment (PatrickMenoti) [reply messages/create/domains/messages.go]: It's both plural and possessive

**Review by PatrickMenoti — DISMISSED** [review]
LGTM

**Review by LucasGeneroso — APPROVED** [review]

**Review by PatrickMenoti — DISMISSED** [review]

**Review by LucasGeneroso — APPROVED** [review]
🚀

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**